components/affix/ng-package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "lib": {
+    "flatModuleFile": "nzAffix",
```

Comment:
Do you think we need the prefix `nz`? Or just set flatModuleFile with `affix`(same with other components) @hsuanxyz

Reply:
As [the source code](https://github.com/ng-packagr/ng-packagr/blob/master/src/ng-package.schema.json#L60) says:
> if empty, defaults to the package name as given in `package.json`

So I think it should be empty for ng-zorro-antd, this is what keeps the behavior consistent with previous versions.